src/js/helpers.js

```diff
@@ -0,0 +1,72 @@
+import def from '../img/default.jpg';
+import day200 from '../img/2xx-day.jpg';
+import day300 from '../img/3xx-day.jpg';
+import day500 from '../img/5xx-day.jpg';
+import day600 from '../img/6xx-day.jpg';
+import day700 from '../img/7xx-day.jpg';
+import day800 from '../img/800-day.jpg';
+import day801 from '../img/80x-day.jpg';
+import night200 from '../img/2xx-night.jpg';
+import night300 from '../img/3xx-night.jpg';
+import night500 from '../img/5xx-night.jpg';
+import night600 from '../img/6xx-night.jpg';
+import night700 from '../img/7xx-night.jpg';
+import night800 from '../img/800-night.jpg';
+import night801 from '../img/80x-night.jpg';
+
+const local = {
+
+  days: {
+    0: 'Sun', 1: 'Mon', 2: 'Tue', 3: 'Wed', 4: 'Thu', 5: 'Fri', 6: 'Sat',
+  },
```

Comment:
Is this an object simulating an array 🤔?

Reply:
Change it =)